README.md

```diff
@@ -1,43 +1,419 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
+# Google Cloud SDK Docker
+
+
+The [Cloud SDK](https://cloud.google.com/sdk/) is the command-line interface to the Google Cloud Platform.  It is a flexible utility which uses GCPs own Cloud APIs to perform many different tasks such as deploying code on AppEngine, to creating Compute Engine VMs to checking IAM permissions and so on.  It is the command line interface to pretty much every Google Cloud Platform API and service.   
+
+This article describes some more ways customers can use it by demonstrating several use cases involving containerizing the Cloud SDK.   The following shows Dockerfiles targeted towards various operating systems and some sample usecases (Its nothing new, just a sample set to document the baseline dockerfiles.
+
+* [Dockerfile](#dockerfile)
+    - [Ubuntu](#ubuntu)
+    - [CentOS](#centos)
+    - [Alpine](#alpine)
+* [Usecases](#usecases)
+    - [Containerize local development environment](#containerize-local-development-environment)
+    - [Run gcloud cli without installing SDK locally](#run-gcloud-cli-without-installing-sdk-locally)
+    - [Run emulators in containers](#run-emulators-in-containers)
+    - [Automate simple DevOps tasklets](#automate-simple-devdps-tasklets)       
+
+## Dockerfile
```

Comment:
@salrashid123 I don't have the data but my instincts tell me that the farthest you can extend this image is to do a `FROM base; RUN gcloud compontents install <...>` and nothing more. If you know of more advanced use cases, please bring them up. Otherwise I think "extending the image" is not even the primary or secondary use case.

You should assume that people extending the image 1) know how to use gcloud 2) know how to write dockerfiles to extend docker images, that's docker 101.